code/modules/reagents/reagents/medicine.dm

```diff
@@ -1501,3 +1501,23 @@
 	metabolism = REM * 0.002
 	overdose = REAGENTS_OVERDOSE * 0.25
 	scannable = 1
+
+/datum/reagent/earthsblood
+	name = "Earthsblood"
+	id = "earthsblood"
+	description = "A rare plant extract with immense, almost magical healing capabilities. Induces a potent psychoactive state, damaging neurons with prolonged use."
+	taste_description = "honey and sunlight"
+	reagent_state = LIQUID
+	color = "#ffb500"
+	overdose = REAGENTS_OVERDOSE * 0.50
+	scannable = 1
+
+/datum/reagent/earthsblood/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	M.adjustBruteLoss (-4 * removed)
+	M.adjustFireLoss (-4 * removed)
```

Comment:
adjustBrute/FireLoss should probably be replaced with heal_organ_damage, similarly to how bicard/derm handle it.

Reply:
Also should consider making earthsblood be affected by per-species chem efficency modifier that used in most medicines.

Reply:
Would this also *heal* organs, though? I don't want it undoing the damage being dealt to one's brain, as the jacked numbers on the chem are only rewarded +because+ it eats away at the brain.

Reply:
heal_organ_damage applies to 'external' organs. Aka hands/arms/torso/etc.

Reply:
heal_organ_damage was put in, I'm thinking about keeping it equally effective across all organics, though.

Reply:
Welcome to unintuitively named procs, we hate it here just as much as you will.

Reply:
As an additional note: the per-species chem modifier isn't actually used by anything that I know of. It's just there as an option that could be used in species or traits. So hooking that up shouldn't change its performance in a significant manner.